addons/l10n_vn/data/account_tax_report_data.xml

```diff
@@ -56,12 +56,24 @@
                                     </record>
                                 </field>
                             </record>
+                            <record id="account_tax_report_line_04_01_01_vn" model="account.report.line">
+                                <field name="name">VAT Exemption on purchase of goods and services</field>
+                                <field name="code">VAT_EXEMPTION_ON_PURCHASE_OF_GOODS_AND_SERVICES</field>
+                                <field name="sequence" eval="4" />
```

Comment:
just a small comment: I don't think we need to specify sequence anymore :slightly_smiling_face: thanks!

Reply:
indeed, if it isn't there it should use the order of declaration.

Reply:
Thank you both, i have updated please review thank you :heart: